From f715ef7f22310e7540397f4dd9de96af29536690 Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Thu, 23 May 2019 19:18:17 +0200
Subject: [PATCH 212/345] watchdog: octeon-wdt: Fix to work on multi-node
 systems.

We need to properly mask the core number when calculating node local
resource offsets.

Signed-off-by: David Daney <david.daney@cavium.com>
Signed-off-by: Carlos Munoz <cmunoz@caviumnetworks.com>
---
 drivers/watchdog/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/watchdog/Kconfig b/drivers/watchdog/Kconfig
index 13b8e3010322..3a74cefe1496 100644
--- a/drivers/watchdog/Kconfig
+++ b/drivers/watchdog/Kconfig
@@ -1509,7 +1509,7 @@ config TXX9_WDT
 	  Hardware driver for the built-in watchdog timer on TXx9 MIPS SoCs.
 
 config OCTEON_WDT
-	tristate "Cavium OCTEON SOC family Watchdog Timer"
+	bool "Cavium OCTEON SOC family Watchdog Timer"
 	depends on CAVIUM_OCTEON_SOC
 	default y
 	select WATCHDOG_CORE
-- 
2.25.1

